cypress/elements/giftCard/giftCardDialog.js

```diff
@@ -1,7 +1,7 @@
 export const GIFT_CARD_DIALOG = {
   amountInput: '[name="balanceAmount"]',
   currencySelectButton: '[id="mui-component-select-balanceCurrency"]',
-  currenciesOptions: '[data-test="selectFieldOption"]',
+  currenciesOptions: '[data-test-id*="select-field-option"]',
```

Comment:
Line 16: 
`cardCodeText: '[data-test-id="card-code"]',`

Reply:
I removed file for gift card - I will fix it when merging to 3.1